apps/yggdrasil/lib/yggdrasil/role.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Yggdrasil.Role do
+  use Ecto.Schema
+
+  schema "roles" do
+    field :name, :string
+    has_many :role_resources, Yggdrasil.RoleResource
```

Comment:
The first parameter is the name right? This usage would be `role.role_resources`? Seems a bit awkward, what about `:permissions` so that the usage is `role.permissions`?